provider/manual/environ.go

```diff
@@ -74,7 +74,7 @@ func (*manualEnviron) StopInstances(context.ProviderCallContext, ...instance.Id)
 
 // AllInstances implements environs.InstanceBroker.
 func (e *manualEnviron) AllInstances(ctx context.ProviderCallContext) ([]instances.Instance, error) {
-	return e.Instances(ctx, []instance.Id{BootstrapInstanceId})
+	return e.Instances(ctx, []instance.Id{instance.Id(fmt.Sprintf("%s%s", BootstrapInstanceId, e.host))})
```

Comment:
Why this change? It's not mentioned in the PR and break unit tests.

Reply:
this was a drive-by that resolved "manual:" machine instance IDs, but I can pull this change our if you think it is appropriate

Reply:
removed this